ios/Sources/AboutFeature/AboutLoadedFeature.swift

```diff
@@ -0,0 +1,81 @@
+import ComposableArchitecture
+import Model
+import UIApplicationClient
+
+public struct AboutLoadedState: Equatable {
+    public var staffs: [Staff]
+    public var contributors: [Contributor]
+    public var selectedType: SelectedType
+    public var aboutDroidKaigiState: AboutDroidKaigiState?
+    public var showingURL: URL?
+
+    public var isShowingSheet: Bool {
+        showingURL != nil || aboutDroidKaigiState != nil
+    }
+
+    public init(
+        staffs: [Staff] = [],
+        contributors: [Contributor] = [],
+        selectedType: SelectedType = .staff
+    ) {
+        self.staffs = staffs
+        self.contributors = contributors
+        self.selectedType = selectedType
+    }
+}
+
+public enum AboutLoadedAction {
+    case selectedPicker(SelectedType)
+    case tapStaff(Staff)
+    case tapContributor(Contributor)
+    case tapBanner
+    case hideSheet
+    case aboutDroidKaigi(AboutDroidKaigiAction)
+}
+
+public struct AboutLoadedEnvironment {
+    public let applicationClient: UIApplicationClientProtocol
+
+    public init(
+        applicationClient: UIApplicationClientProtocol
+        ) {
```

Comment:
weird indent?

Reply:
Thank you. Deleted in d963558990a5e9b93158a74cc18ca2df489c5de9